eo-runtime/src/main/eo/org/eolang/tuple.eo

```diff
@@ -57,10 +57,16 @@
         index.lt 0
         index.gte len
       error "Given index is out of tuple bounds"
-      if.
-        index.lt (len.plus -1)
-        ^.head.at index
-        ^.tail
+      ^.at.at-without-checks index
+
+    # Takes element from the tuple without checking i.
+    # i must be less than length and greater or equal 0.
```

Comment:
@levBagryansky `i` actually may be any object, it's not mandatory `i` to be `int`. It can be for example
```
[] > my-i
  [x] > lt
    TRUE > @
```
And it will work. So I would say that it's expected `i` has `lt` attribute and we don't check its value

Reply:
@maxonfjvipon i should be int. Otherwise the object does not have sense

Reply:
@levBagryansky It should not and we can't guarantee it. All we need that `i` has attribute `lt` with 1 free attribute. That's it

Reply:
What will be `(* 1 2 3) .at my-int` here? Not ub but something like that, because then the method guarantees nothing

Reply:
@maxonfjvipon Also I did not write that `i` should be `int`. i must be less than length and greater or equal 0. That is all. If you object is less  than length and greater or equal to 0 then you can use this method.

Reply:
@levBagryansky `lt` means "less than" only because we decided to attach that semantics to the attribute. What if `i` is an object with just `lt` and `gte` attribute that are going to google and check the weather on the Mars.
I'm trying to say that we should not get attached to semantics. I believe description should just describe the final behaviour

Reply:
When I write x less than y, I mean `x.lt y` is TRUE. Ok, we can write the condition this way: i.lt len is TRUE

Reply:
@levBagryansky but `x.lt y` is not supposed to mean "x is less than y"

Reply:
Supposed to be. Actually it is abstraction. We have polymorphism here.

Reply:
We can remove it if it seems confusing

Reply:
@levBagryansky I would propose something like this for the description: 
Takes element from the tuple without index validation.
Here `i` must be an object that can be comparable with `int` using `lt` method.

Reply:
@maxonfjvipon ok